src/main/webapp/app/course/manage/course-update.component.html

```diff
@@ -336,6 +336,26 @@ <h5>
                             ngbTooltip="{{ 'artemisApp.course.courseCommunicationSetting.messagingEnabled.tooltip' | artemisTranslate }}"
                         ></fa-icon>
                     </div>
+                    <div class="form-group" *ngIf="messagingEnabled">
+                        <label
+                            class="form-control-label"
+                            jhiTranslate="artemisApp.course.courseCommunicationSetting.messagingEnabled.codeOfConduct.label"
+                            for="field_messagingCodeOfConduct"
+                            >Messaging Code of Conduct
+                        </label>
+                        <fa-icon
+                            [icon]="faQuestionCircle"
+                            class="text-secondary"
+                            ngbTooltip="{{ 'artemisApp.course.courseCommunicationSetting.messagingEnabled.codeOfConduct.tooltip' | artemisTranslate }}"
+                        ></fa-icon>
```

Comment:
We have the jhi-help-icon component for that

Reply:
I'll use that